src/data.cpp

```diff
@@ -166,7 +173,7 @@ double pyne::natural_abund(int nuc) {
   // state abundance...not strictly true, but good guess.
   if (0 < nucid%10000) {
     na = natural_abund((nucid/10000)*10000);
-    atomic_mass_map[nuc] = na;
+    natural_abund_map[nuc] = na;
```

Comment:
Yeah, but I think we should have a different PR for all the `natural_abund_map` stuff.